ui/perf.html

```diff
@@ -40,6 +40,7 @@
         <ul class="dropdown-menu icon-menu" role="menu">
           <li><a ui-sref="dashboard({topic: 'e10s'})"><span class="fa fa-bar-chart midgray"></span> e10s</a></li>
           <li><a ui-sref="e10s_trend"><span class="fa fa-line-chart midgray"></span> e10s-trend</a></li>
+          <li><a ui-sref="dashboard({topic: 'hasal', repo: 'mozilla-central'})"><span class="fa fa-bar-chart midgray"></span> Hasal</a></li>
```

Comment:
Nit: For the menu text please use lower case "hasal" instead of "Hasal", so it is consistent with the other Dashboard sub-menu items.